components/layout/CloudBanner.tsx

```diff
@@ -3,31 +3,42 @@ import Link from 'next/link'
 import { IconRight } from '../blocks'
 import { Button, ButtonGroup } from '../../components/ui'
 import GitHubButton from 'react-github-btn'
-
+import { EmailForm } from '../forms/EmailForm'
 export function CloudBanner() {
   return (
     <>
-      <div className="banner">
-        <div className="banner-content w-full flex justify-center items-center">
-          <span className="desktop">
-            <p className="mr-5">Loving Tina? ⭐️ us on <a href="https://github.com/tinacms/tinacms" target="new">GitHub</a></p>
-          </span>
-          {/* @ts-ignore */}
-          <GitHubButton
+    
+    <div className="banner">
+    <div className="banner-content flex justify-center items-center banner-left-padding">
+        <span className="mobile">
+            <p className="mr-5">
+                Loving Tina? ⭐️ us on{' '}
+                <a href="https://github.com/tinacms/tinacms" target="new">
+                    GitHub
+                </a>
+            </p>
+        </span>
+        {/* @ts-ignore */}
+        <GitHubButton
             href="https://github.com/tinacms/tinacms"
             data-size="medium"
             data-show-count="true"
             aria-label="Star TinaCMS on GitHub"
-          >
+        >
             Star
-          </GitHubButton>
-        </div>
-      </div>
+        </GitHubButton>
+    </div>
+    <div className="banner-content w-auto flex justify-center">
+        <div className="mr-1 mt-1 justify-top"><p>Stay in touch 👉</p></div>
+        <span><EmailForm isFooter /></span>
```

Comment:
Do we need a span wrapping this form element? It seems to flex as intended without it

Reply:
No, we don't need it. It looks like it was there in the original code and when I moved the component to the top, it came with the code chunk. Removed it.